.clang-format

```diff
@@ -0,0 +1,9 @@
+---
+BasedOnStyle: Mozilla
+---
+Language: Cpp
+AllowAllParametersOfDeclarationOnNextLine: true
+PointerAlignment: Right
+IndentWidth: 4
```

Comment:
But then it's not `Mozilla` style...what other projects use this style anyways?  I had assumed it was from PackageKit but AFAICS that project is all `linux` i.e. `/* -*- Mode: C; tab-width: 8; indent-tabs-mode: t; c-basic-offset: 8 -*-
` (in Emacs terms).

Personally I'd prefer either GNU or Linux style; most projects I interact with are one or the other.